src/test/java/com/training/api/tests/ga/GuestAccountTests.java

```diff
@@ -65,4 +71,80 @@ public void testLoginAuthentication(){
         softAssertions.assertThat(responseElement.getPayload().isUid()).as("Email doesn't exists")
                 .isTrue();
     }
+
+    @Test
+    public void testNegativeLoginValidationWrongAppKey() {
+        String errorCode = "COMMONS-0001";
+        headerMap.put(map.get("gaValidationAppKeyHeaderName"), map.get("gaValidationAppKeyHeaderWrongValue"));
+        com.training.pojos.ga.validation.Request request = new Request();
+        request.setPassword(PASSWORD);
+        request.setUid(EMAIL);
```

Comment:
put this lines of code in before each and do changes in @test based on the requirement

Reply:
for one test request is from a different class, i cant use same for all